compiler/sigmatch.nim

```diff
@@ -568,7 +568,7 @@ proc inconsistentVarTypes(f, a: PType): bool {.inline.} =
 
 proc procParamTypeRel(c: var TCandidate, f, a: PType): TTypeRelation =
   ## For example we have:
-  ##..code-block:: nim
+  ##.. code-block:: nim
```

Comment:
still incorrect; `nimlint` should report this.

this works:
```
  ## For example we have:
  ##
  ## .. code-block:: nim
  ##   proc myMap[T,S](sIn: seq[T], f: proc(x: T): S): seq[S] = ...
  ##   proc innerProc[Q,W](q: Q): W = ...
```

as can be seen by exporting the symbol and `nim doc -r compiler/sigmatch.nim`

`nim doc -r --docInternal compiler/sigmatch.nim`

yes, `--docInternal` is nifty.

Reply:
Indeed.